src/Reflex/Dom/Xhr.hs

```diff
@@ -50,6 +50,9 @@ import Data.Map (Map)
 import qualified Data.Map as Map
 import Data.Maybe
 import Data.Text (Text)
+import qualified Data.Text as T
+import GHCJS.DOM.XMLHttpRequest (getAllResponseHeaders)
```

Comment:
That's a really good point.  We need to make sure this feature is compatible with both platforms.  @mageshb Do you think it would be better to add getAllResponseHeaders to ghcjs-dom (ghc version) or to use getResponseHeader instead?